ibm_catalog.json

```diff
@@ -0,0 +1,460 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-cloud-logs",
+        "label": "Cloud automation for Cloud Logs",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "logging_monitoring"
+        ],
+        "keywords": [
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "cloud logs",
+          "observability",
+          "logging",
+          "monitoring",
+          "monitor",
```

Comment:
Remove "monitoring" and "monitor". Thats a different service: https://github.com/terraform-ibm-modules/terraform-ibm-cloud-monitoring